CloudFormation/Docker/windows-dotnet-web-application/README.md

```diff
@@ -0,0 +1,110 @@
+# Windows Server 2019 Container Image Pipeline for hosting a .NET web application
+
+This is a sample template that demonstrates how to use EC2 Image Builder CloudFormation resources to build an Windows Server 2019 Docker container image that can host a .NET web application. The image will be published to the specified Amazon Elastic Container Registry (ECR) repository.
```

Comment:
`an` -> `a`: "...to build a Windows Server..."

(only commented on this as there's a tag that should be updated as well)